cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/api/ClusterBackupController.java

```diff
@@ -0,0 +1,117 @@
+package io.mesosphere.mesos.frameworks.cassandra.scheduler.api;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import io.mesosphere.mesos.frameworks.cassandra.CassandraFrameworkProtos;
+import io.mesosphere.mesos.frameworks.cassandra.scheduler.CassandraCluster;
+import io.mesosphere.mesos.frameworks.cassandra.scheduler.util.ClusterJobUtils;
+import io.mesosphere.mesos.frameworks.cassandra.scheduler.util.JaxRsUtils;
+import org.jetbrains.annotations.NotNull;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+
+@Path("/cluster/backup")
+@Produces("application/json")
+public final class ClusterBackupController {
+
+    @NotNull
+    private final CassandraCluster cluster;
+    @NotNull
+    private final JsonFactory factory;
+
+    public ClusterBackupController(@NotNull final CassandraCluster cluster, @NotNull final JsonFactory factory) {
+        this.cluster = cluster;
+        this.factory = factory;
+    }
+
+    /**
+     * Starts a cluster-wide backup.
+     *
+     *     Example: <pre>{@code {
+     * "started" : true
+     * }}</pre>
+     */
+    @POST
+    @Path("/start")
+    public Response backupStart(@QueryParam("name") String name) {
+        if (name == null || name.isEmpty()) {
+            return JaxRsUtils.buildStreamingResponse(factory, new StreamingJsonResponse() {
```

Comment:
This builder will return a `200` http status code, this should be in the 400 range since it's a client error. Use [`io.mesosphere.mesos.frameworks.cassandra.scheduler.util.JaxRsUtils#buildStreamingResponse(JsonFactory, Status, StreamingJsonResponse)`](https://github.com/mesosphere/cassandra-mesos/blob/master/cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/util/JaxRsUtils.java#L127) instead.